plugins/woocommerce/client/blocks/assets/js/atomic/blocks/product-elements/product-image-gallery/upgrade-notice.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import { dispatch, select } from '@wordpress/data';
+import { UpgradeDowngradeNotice } from '@woocommerce/editor-components/upgrade-downgrade-notice';
+import { findBlock } from '@woocommerce/utils';
+import { createBlock } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+
+const upgradeToBlockifiedProductGallery = ( blockClientId: string ) => {
+	const blocks = select( 'core/block-editor' ).getBlocks();
+	const foundBlock = findBlock( {
+		blocks,
+		findCondition: ( block ) =>
+			block.name === metadata.name && block.clientId === blockClientId,
+	} );
+
+	if ( foundBlock ) {
+		const newBlock = createBlock( 'woocommerce/product-gallery' );
+		dispatch( 'core/block-editor' ).replaceBlock(
+			foundBlock.clientId,
+			newBlock
+		);
+	}
+};
+
+export const UpgradeNotice = ( {
+	blockClientId,
+}: {
+	blockClientId: string;
+} ) => {
+	if ( ! blockClientId ) {
+		return null;
+	}
+
+	const notice = createInterpolateElement(
+		__(
+			'Gain access to more customization options when you upgrade to the <strongText />.',
+			'woocommerce'
+		),
+		{
+			strongText: (
+				<strong>
+					{ __( `blockified experience`, 'woocommerce' ) }
```

Comment:
“blockified” is a term we have been using internally, and now it's ingrained so much in us that we don't even question the meaning. While it seems pretty intuitive, I have had other people from A8C/Woo being confused by this term.

I'd honestly ask for this copy to be reviewed by someone… not sure whom, but perhaps @woocommerce/docs-review ?

Reply:
This text comes from DSGWOO-268. I agree we could update it to avoid using the 'blockified' word, though. Maybe simply replacing `blockified` with `new` would work in all cases? :thinking: 

In any case, I would avoid blocking this PR because of that, considering this string is already in the released versions:

https://github.com/woocommerce/woocommerce/blob/755c75d2144e72f81cc789c65ca7bdc155e1a7a4/plugins/woocommerce/client/blocks/assets/js/blocks/product-elements/add-to-cart-form/components/upgrade-notice/index.tsx#L53

Reply:
I came here to say the same. "blockified" may not make sense outside of a8c context, so let's follow-up and fix this. 

Agreed with @Aljullu here: we don't necessarily have to decide on how to call "blockified" publicly, we can just use "new" or "recommended".